src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -235,10 +189,6 @@ const DropDownMenu = React.createClass({
    */
   getInputNode() {
```

Comment:
> I tried to remove this method once. It bit me so hard I spent a day in hospital

:laughing: :laughing: I hear you with that.

The only reason why I ask is because I do see a separate `_getInputNode()` method in `TextField.js`. I'm not suggesting we remove that one. But I've search the code base and I just can't seem to find where the `getInputNode()` method is being used.

Let me know!